tests/system/providers/elasticsearch/example_elasticsearch_python_hook.py

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Rename the file example_elasticsearch_hook.py ?

Reply:
I've merged the new example into the old examples file :) 
Now it should give the example to both hooks